payments/management/commands/send_sap_invoices.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import paramiko
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.utils import timezone
+
+from payments.models import Invoice
+
+LOG = logging.getLogger(__name__)
```

Comment:
Why not use `Command.stdout` or `Command.stderr` here instead of a logger?

Reply:
I actually wanted to keep the `upload_xml_to_sftp` function outside the Command class so it could be invoked without having to use the management command.

Reply:
Maybe then can be added to the app package rather than command so easier to import.

Reply:
I moved it inside the management command since I don't think we'll need to import elsewhere. I also replaced the logging with `Command.stdout`.